qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -1635,23 +1685,33 @@ def config(self) -> Dict[str, Any]:
         }
 
     @classmethod
+    @deprecate_func(
+        since="0.6",
+        additional_msg="This method will be removed and no alternative will be provided.",
+        package_name="qiskit-experiments",
+    )
```

Comment:
Why? Seems a bit brutal. The alternative is to serialize through the Encoder/Decoder no? Can you state that?

Reply:
Thanks. Updated in [Documentation update](https://github.com/Qiskit/qiskit-experiments/pull/1120/commits/5ac45190fa015b34ba7ec2056c15e552fa1bb5d6)